Onepunch/README.md

```diff
@@ -1,18 +1,38 @@
-# Onepunch
+# Onepunch by Saitama D. Luffy
```

Comment:
It's better to keep the title of the README only with the theme title, as specified in the main README, in the contributions section